### ISSUE TITLE
Smarter units for bandwidths

### ISSUE THREAD

**Issue by skyreflectedinmirrors** [ROCm/rocprofiler-compute#145]
It would be nice if we could get units for bandwidth values that scale with the size.  E.g., if we have > a GB/s, label it GB/s, or MB/s or KB/s, etc.

Not sure how possible this actually is in the current design, but would improve readability significantly.

cc: @feizheng10

**Comment by feizheng10** [comment]
Yes,  technically it is not hard to do for single run...a bit tricky for multiple runs with comparison.
I would suggest to do on tty/gui level.

**Comment by amd-hsivasun** [comment]
Imported to ROCm/rocm-systems
